src/beforeBuild.ts

```diff
@@ -15,12 +15,28 @@
  */
 
 import { cp, mkdir, readFile, rm, writeFile } from "node:fs/promises";
-import { join } from "node:path";
+import { join, resolve } from "node:path";
 
 import { compileFromPath } from "@fluencelabs/aqua-api";
 import aquaToJs from "@fluencelabs/aqua-to-js";
-
-import { AQUA_EXT, FS_OPTIONS } from "./lib/const.js";
+import { gatherImportsFromNpm } from "@fluencelabs/npm-aqua-compiler";
+
+import {
+  AQUA_DEPENDENCIES_DIR_NAME,
+  AQUA_EXT,
+  FS_OPTIONS,
+  NODE_MODULES_DIR_NAME,
+} from "./lib/const.js";
+import { versions } from "./versions.js";
+
+const aquaDependenciesDirPath = join("src", AQUA_DEPENDENCIES_DIR_NAME);
+await mkdir(aquaDependenciesDirPath, { recursive: true });
+
+await writeFile(
+  join(aquaDependenciesDirPath, "package.json"),
+  JSON.stringify({ dependencies: versions.npm }, null, 2),
```

Comment:
I thought you had a function for this `jsonStringify`

Reply:
Or you don't want to import something from CLI in this helper script?

Reply:
yeah don't want to import that, it's the reason